tokens/oin.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ethereum_address": "0x9aeB50f542050172359A0e1a25a9933Bc8c01259",
+    "aurora_address": "",
+    "name": "OIN Finance",
```

Comment:
It's `"oinfinance"`, not `"OIN Finance"` on Ethereum. Should be the same, please change.

Reply:
ok ,fixed